### ISSUE TITLE
Feature "Unexploded query params" is not supported.

### ISSUE THREAD

**Issue by barnard-b** [apple/swift-openapi-generator#52]
Using an Open API file that has a parameter with `explode: false` results in a warning of
```
Feature "Unexploded query params" is not supported, skipping [context: foundIn=Operations.launcher_list.Input.Query (#/paths/2.2.0/launcher/GET/query)/launcher_config__ids]
```

`false` values appear to be listed as unsupported in the documentation.
<img width="489" alt="image" src="https://github.com/apple/swift-openapi-generator/assets/44757473/fa2ebc63-a7e4-49c8-9746-6adc526dc913">

```yaml
  /launcher/:
    get:
      operationId: launcher_list
      parameters:
        - in: query
          name: launcher_config__ids
          schema:
            type: array
            items:
              type: integer
          description: Comma-separated launcher config IDs.
          explode: false
          style: form
      responses:
        "200":
          description: A success response with a greeting.
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Greeting"
```

I am open to creating a PR, but this is my first time using Open API and would need guidance.

**Comment by czechboy0** [comment]
Hi @barnard-b, thanks for reporting this! Indeed, `explode: false` isn't supported yet. The issue https://github.com/apple/swift-openapi-generator/issues/18 covers both `explode: false` and other styles, but we can keep this issue to only track adding support for `explode: false`.

We would definitely appreciate a PR! To learn more about contributing, check out these docs: https://swiftpackageindex.com/apple/swift-openapi-generator/0.1.0/documentation/swift-openapi-generator/contributing-to-swift-openapi-generator

And if you have any questions, don't hesitate to tag me here 🙂

**Comment by czechboy0** [comment]
This will be released shortly in 0.1.8.
